CanonicalLtd/documentation-builder
# Add highlight.js styles

QA
---

``` bash
python3 -m venv env3  # Prepare an environment
env3/bin/pip install -e .  # Install this version of documentation-builder
curl https://pastebin.com/raw/gcG2rPs0 > docs/codeblocks.md  # Download the test file
env3/bin/python bin/documentation-builder --source-folder docs  # Build the HTML pages
```

Open up `build/en/codeblocks.html`. The two code blocks should both be highlighted, and look more-or-less the same.
